staging/src/k8s.io/api/resource/v1alpha3/register.go

```diff
@@ -52,6 +52,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&ResourceClaimTemplateList{},
 		&ResourceSlice{},
 		&ResourceSliceList{},
+		&DeviceTaintRule{},
```

Comment:
No equivalent for v1beta2?

Reply:
It's an alpha-quality API. I suppose we don't want to make it available in the beta version?

Reply:
oh, of course ;)  Sorry.